tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -278,6 +278,10 @@ static cl::opt<SPIRV::BuiltinFormat> SPIRVBuiltinFormat(
         clEnumValN(SPIRV::BuiltinFormat::Global, "global",
                    "Use globals to represent SPIR-V builtin variables")));
 
+static cl::opt<bool>
+    NoEntry("no-entry", cl::init(false),
```

Comment:
For consistency with other llvm-spirv options:

```suggestion
    NoEntry("spirv-no-entry", cl::init(false),
```